### PR TITLE
Updated Swedish translation

### PR DESCRIPTION
**Pull request by hanklank**
Updated Swedish translation.

### CODE REVIEW

File: po/sv.po [hunk 1]
```diff
@@ -2001,21 +1963,113 @@ msgstr "Misslyckades med att synkronisera med dbus-proxy"
...
+#. - Normal users do not require admin authentication to update
+#. appstream data as it will be signed, and the action is required
+#. to update the system when unattended.
+#. - Changing this to anything other than 'yes' will break unattended
+#. updates.
+#: system-helper/org.freedesktop.Flatpak.policy.in:138
+msgid "Update appstream"
+msgstr "Uppdatera appstream"
+
+#~ msgid "OAutomatically answer yes for all questions"
```
Comment (AsciiWolf) [inline po/sv.po]: Why the unnecessary comment? (It was a typo that is fixed and won't appear in the code anymore.)

Comment (hanklank) [inline po/sv.po]: Old translation is per default commented - that's just how most of the translation community does its work per default. Look in any po-file with a history. Examples from reallife: flatpak german https://l10n.gnome.org/POT/flatpak.master/flatpak.master.de.po, appstream hungarian https://l10n.gnome.org/POT/appstream.master/appstream.master.hu.po, Croatian gstreamer https://l10n.gnome.org/POT/gstreamer.master/gstreamer.master.hr.po and so on and so on.

Comment (AsciiWolf) [inline po/sv.po]: @hanklank Yes, I know, but as I said, this one was a typo, not some old, now unused translatable string. I really don't see a point of having it as a comment.

Comment (hanklank) [inline po/sv.po]: Sure, I'll remove the old commented string, it's not important in anyway. Gone.

**Pull request merged**